### PR TITLE
[DOCS] Adds info about custom dashboards to dashboards landing page doc 

### LINKED ISSUE #2277: [DOCS] Custom dashbaords in Elastic Security
## Description 

Users can now create custom dashboards from the Security app and list them in the Dashboards landing page in Security.

Related:
- Issue: https://github.com/elastic/kibana/issues/135778
- PR: https://github.com/elastic/kibana/pull/136221
- PR: https://github.com/elastic/kibana/pull/136671

### Sample image
image

### PR DESCRIPTION
**Pull request by benironside**
Fixes #2277 by adding information about custom dashboards to the dashboards landing page. Let me know if you think there's anything else we should tell users about this.

[Preview](https://security-docs_2533.docs-preview.app.elstc.co/guide/en/security/master/dashboards-overview.html)

### CODE REVIEW

File: docs/dashboards/dashboards-overview.asciidoc [hunk 1]
```diff
@@ -4,6 +4,13 @@
 
 The following sections describe the {security-app}'s prebuilt dashboards, which provide visualizations of your security environment.
 
+You can also create and view custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard** in the upper-right corner. Once created, custom dashboards appear under CUSTOM at the bottom of the page:
```
Comment (joepeeples) [inline docs/dashboards/dashboards-overview.asciidoc:7]: All-caps CUSTOM feels shouty -- I know our [style guide](https://docs.elastic.dev/content-design/tech-docs/writing-about-ui#general-guidelines-for-writing-about-the-ui) says to reproduce the capitalization from the UI, but I think that's for Title Case or Sentence case, not all-caps. Maybe change to "Custom" (and bold?), or consider omitting entirely -- we may not need to name the section, just say where it is:
```suggestion
You can also create and view custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard** in the upper-right corner. Once created, custom dashboards appear at the bottom of the page:
```

**Review by joepeeples — COMMENTED** [review]
One suggestion for consideration, looks good!

File: docs/dashboards/dashboards-overview.asciidoc [hunk 2]
```diff
@@ -4,6 +4,13 @@
 
 The following sections describe the {security-app}'s prebuilt dashboards, which provide visualizations of your security environment.
 
+You can also create and view custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard** in the upper-right corner. Once created, custom dashboards appear at the bottom of the page:
```
Comment (nastasha-solomon) [inline docs/dashboards/dashboards-overview.asciidoc:7]:
```suggestion
You can also create and access custom security dashboards from the Dashboards landing page. To create one, click **Create Dashboard**. Once created, custom dashboards appear on the page:
```

Comment (nastasha-solomon) [reply docs/dashboards/dashboards-overview.asciidoc:7]: Made some slight modifications based on our guidelines for writing for accessibility which recommend avoiding directional language. 
https://docs.elastic.dev/content-design/accessibility

(I sometimes forget about these guidelines so I'm sure I'll be giving myself the same edits soon ;) )

**Review by nastasha-solomon — APPROVED** [review]
Made some light edits. Otherwise, LGTM!

File: docs/dashboards/dashboards-overview.asciidoc (hunk 2 above)
Comment (jmikell821) [reply docs/dashboards/dashboards-overview.asciidoc:7]: Accessibility is tricky. Tbh, sometimes I think it's necessary, especially in a complex or "noisy" UI, but understand why there are guidelines in there to suggest against directional language. You could also say something like `"Once created, confirm that your custom dashboard appears on the page." `

**Review by jmikell821 — COMMENTED** [review]
Just a comment/alt suggestion. LGTM! 👍

**Pull request merged**